pkg/assembler/backends/ent/backend/search.go

```diff
@@ -310,12 +310,41 @@ func (b *EntBackend) BatchQueryPkgIDCertifyVuln(ctx context.Context, pkgIDs []st
 		queryList = append(queryList, convertedID)
 	}
 
-	var predicates []predicate.CertifyVuln
+	var cvLatestScan []struct {
+		PkgID          uuid.UUID `json:"package_id"`
+		VulnID         uuid.UUID `json:"vulnerability_id"`
+		LastScanTimeDB time.Time `json:"max"`
+	}
+
+	var aggPredicates []predicate.CertifyVuln
+	aggPredicates = append(aggPredicates, certifyvuln.PackageIDIn(queryList...), certifyvuln.VulnerabilityIDNEQ(noVulnID))
+
+	// aggregate to find the latest timescanned for certifyVulns for list of packages
+	err := b.client.Debug().CertifyVuln.Query().
```

Comment:
I suppose `Debug()` supported the development but it's worth removing.
```suggestion
	err := b.client.CertifyVuln.Query().
```

Reply:
ah yes i meant to remove that!